### PR TITLE
Add linters html css

### PR DESCRIPTION
**Pull request by 29td**
Added linters to correct my codes
Added lighthouse to improving the quality of web pages.
Added Webhint to improve your site's accessibility.
Added Stylelint to avoid errors and enforce conventions in your styles.

### CODE REVIEW

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
.

File: README.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-- Frameworks
-- Technologies used
+- HTML/CSS
+- Node.js
+
 
 ## Live Demo (if available)
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](127.0.0.1:5500/index.html)
```
Comment (vechicin) [inline README.md]: I suggest you remove this section altogether if you don't plan on deploying a live demo.

File: README.md [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
 
 
 ## Getting Started
```
Comment (vechicin) [inline README.md:15]: I suggest you edit this section so you can add instructions on how to get a local copy of this project and how to install the necessary dependencies for it to run properly.

File: README.md [hunk 3]
```diff
@@ -42,15 +42,9 @@ To get a local copy up and running follow these simple example steps.
 
 👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
+- GitHub: [@githubhandle](https://github.com/29td)
+- Twitter: [@twitterhandle](https://twitter.com/NonhlanhlaMnde6)
```
Comment (vechicin) [inline README.md]: I recommend you change `Author1` to your full name, it'll make your repo more professional. I'd also recommend you add your LinkedIn profile if you have one, that's the first thing recruiters and potential employers will look for when trying to contact you.

File: README.md [hunk 4]
```diff
@@ -42,15 +42,9 @@ To get a local copy up and running follow these simple example steps.
...
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
 
 ## 🤝 Contributing
```
Comment (vechicin) [inline README.md:49]: I recommend you change the `Acknowledgments` section to actually reflect some acknowledgments. You can thank Microverse or say this was a Microverse project.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Hello Microverse!</h1>
+</body>
+</html>
```
Comment (vechicin) [inline index.html]: Your HTML file doesn't have a CSS file attached to it, this is the reason why your header is not colored gold. Kindly fix this as it is a [project requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md).

File: style.css [hunk 6]
```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: gold;
+}
```
Comment (vechicin) [inline style.css]: [OPTIONAL] Using color names is considered bad practice. I recommend you change this color to its `rgb` or `hex` equivalent.

**Comment by 29td** [comment]
Hello
@Omar Muhammad

I've made the changes below is my link
https://github.com/29td/Hello-world/pull/1

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @29td  ✋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. 🚨 


 ## Highlights ⚡ 

- [x] Correct GitHub Flow
- [x] Linters checks passing
- [x] Excellent use of `.gitignore` file
- [x] Good PR title/description
- [x] Good commit history 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 6 above)
Comment (29td) [reply style.css]: @vechicin 
hey changes were made below is my link:
https://github.com/29td/Hello-world/pull/1

Comment (vechicin) [reply style.css]: @29td Don't forget to request a second review through your dashboard.

**Comment by 29td** [comment]
***@***.****
*i've made changes *https://github.com/29td/Hello-world/pull/1

**Comment by 29td** [comment]
@Omar Muhammad
Greetings I've made changes below is my GitHub url:
https://github.com/29td/Hello-world/pull/1

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @29td  🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by 29td** [comment]
Awesome, thanks!

**Pull request merged**